### PR TITLE
[IMP] website: improve `s_banner` snippet

### PR DESCRIPTION
**Pull request by anso-odoo**
The goal of this PR is to improve the visual design of the default `s_banner` snippet.

requires: https://github.com/odoo/design-themes/pull/866

task-4109584

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by anso-odoo** [comment]
@stefanorigano Adding `s_blockquote` in `s_banner`. Can you have a look ?

File: addons/website/views/snippets/s_banner.xml [hunk 1]
```diff
@@ -18,8 +18,23 @@
                 <div class="o_grid_item o_grid_item_image g-col-lg-2 g-height-5 col-lg-2 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 3; grid-area: 2 / 11 / 7 / 13;">
                     <img class="img img-fluid mx-auto rounded" src="/web/image/website.s_banner_default_image_3" alt=""/>
                 </div>
-                <div class="o_grid_item o_grid_item_image g-col-lg-3 g-height-5 col-lg-3 d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 4; grid-area: 7 / 7 / 12 / 10;">
-                    <img class="img img-fluid mx-auto rounded" src="/web/image/website.s_banner_default_image" alt=""/>
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_banner.xml:22]: Doing this way we'll keep `website .s_banner_default_image` defined, but we'll never use it 😉
Use it for the blockquote (L29 c.a.), eg:

```xml
<img src="/web/image/website.s_banner_default_image" class="s_blockquote_avatar img rounded-circle" alt=""/>
```

... and,  in `image_library.xml`, change the `url` attribute  of `s_banner_default_image` to the desired one -> `/website/static/src/img/snippets_demo/s_team_member_2.jpg`

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Thanks @anso-odoo , some remarks for you 👍
Don't forget the mobile adaptations 😉

**Review by stefanorigano — APPROVED** [review]
LGTM, thanks @anso-odoo 👍

**Comment by sobo-odoo** [comment]
@anso-odoo I pushed [this small diff](https://github.com/odoo/odoo/compare/f076b05f44b7a068f97097baecb95dfaf6f4625c..c3bbcc07b4d8024fcab93032e8a3a91cc7188088) to:
- limit the commit message at 72 chars
- remove the `justify-content-center` class from the text grid item => already there before but it had no effect anyway since the display is not flex, so better remove it :+1: 
- add the `data-name="Blockquote"` attribute, so the blockquote does not just say "block" :wink:

**Review by sobo-odoo — APPROVED** [review]
It's okay with me :wink: 

@robodoo r+

**Pull request closed without merging**